app/controllers/api/auth/stories_controller.rb

```diff
@@ -8,7 +8,7 @@ class Api::Auth::StoriesController < Api::StoriesController
 
   filter_resources_by :account_id, :series_id, :network_id
 
-  filter_params :highlighted, :purchased, :v4, :text, :noseries
+  filter_params :highlighted, :purchased, :v4, :text, :noseries, :state
```

Comment:
I opted to call this `state` and give it a single value.  The alternative would be completely separate filters like:

```
?filters=published
?filters=scheduled
?filters=draft
```

Open to suggestions on which is better.